README.md

```diff
@@ -45,6 +46,7 @@ output_path
 │   └── transformation_2.cpp
 ...
 ```
+Here `log.txt` file contains data on what errors appeared after applying each transformation from `config.yaml`.
```

Comment:
Can you provide more detailed information about the log format? Maybe with example.